postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -421,12 +450,21 @@ def _collect_plan_for_statement(self, row):
                         event['timestamp'] = get_timestamp(row['state_change']) * 1000
             return event
 
-    def _explain_pg_stat_activity(self, rows):
+    def _collect_events(self, rows):
+        active_query_rows = []
```

Comment:
collecting both types of events in a single method allows us to iterate the rows only twice. once for `_filter_valid_statement_rows` and once here